internal/brokers/examplebroker/broker.go

```diff
@@ -493,7 +493,7 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, sessionInfo sessionI
 
 	case "phoneack1":
 		// TODO: should this be an error rather (not expected data from the PAM module?
-		if authData["wait"] != "true" {
+		if authData["wait"] != requests.WaitSet {
```

Comment:
These changes have nothing to do with the PR subject. You either move them to a separate one and then we'll discuss them there or we drop it entirely.

Reply:
Well, they do to me since it's related to "parse encoded values", but if you prefer I'll move them to another PR.